urts/sys/Cargo.toml

```
@@ -1,21 +1,28 @@
 [package]
 name = "mc-sgx-urts-sys"
 version = "0.1.0"
+authors = ["MobileCoin"]
+categories = ["external-ffi-bindings", "hardware-support"]
+description = "FFI bindings for `sgx_urts`."
 edition = "2021"
+keywords = ["ffi", "sgx"]
+license = "Apache-2.0"
```

Do we want links for the `so`s?
The docs, https://doc.rust-lang.org/cargo/reference/build-scripts.html#the-links-manifest-key, just say "native library", which can be either.
```suggestion
license = "Apache-2.0"
links = "sgx_urts"
```